srcpkgs/bruno/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'bruno'
+pkgname=bruno
+version=2.1.0
+revision=1
+archs="x86_64"
+hostmakedepends="n python3 python3-distutils-extra"
+short_desc="Fast and Git-Friendly Opensource API client"
+maintainer="Orphaned <orphan@voidlinux.org>"
+license="MIT"
+homepage="https://www.usebruno.com/"
+changelog="https://www.usebruno.com/changelog"
+distfiles="https://github.com/usebruno/bruno/archive/refs/tags/v${version}.tar.gz"
+checksum=f9176d935940905b2ccb9083e3345332efe56c931146965c2c880a0a89d3d0a0
+
+do_build() {
+	# using node 20
+	n 20
+
+	# dependencies
+	npm run setup
+
+	# node gyp dependencies
+	npm install node-addon-api
+
+	# actual bruno
+	npm run build:web
+	npm run build:electron
+}
+
+do_install() {
+	local package_location="usr/lib/$pkgname"
+
+	vlicense license.md
+
+	vmkdir ${package_location}
+	vcopy packages/bruno-electron/out/linux-unpacked/* ${package_location}
+
+	vinstall ${FILESDIR}/bruno.desktop 644 usr/share/applications
+	vinstall assets/images/logo.png 644 usr/share/pixmaps bruno.png
+
+	vmkdir usr/bin
+	ln -sfr $DESTDIR/${package_location}/bruno $DESTDIR/usr/bin/bruno
+}
```

Comment:
This is a heavily customized build process which only supports `x86_64`. I am not knowledgeable of Node.js nor npm, so I cannot review this, but be aware that you might face more scrutiny from the Void maintainers because of this. They may be less willing to merge this package.

Reply:
That's fine for me if we do not merge.
My follow up question is, do you prefer I close the PR?